### PR TITLE
Release notes for percona agent 1.0.10

### PR DESCRIPTION
**Pull request by hrvojem**
docs preview: http://cloud-docs.percona.com/en/rn-1.0.10/release-notes/1.0/1.0.10.html

### CODE REVIEW

File: doc/release-notes/1.0/1.0.10.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+======================
+ Percona Agent 1.0.10
+======================
+
+Percona is glad to announce the release of Percona Agent 1.0.10 on December 30th 2014. Downloads are available from our download site `here <http://www.percona.com/downloads/percona-agent/1.0.10>`_. Percona Agent can be `installed <http://cloud-docs.percona.com/Install.html>`_ or `upgraded <http://cloud-docs.percona.com/Install.html#updating-the-agent>`_ as described in the `manual <http://cloud-docs.percona.com/index.html>`_. 
+
+New Features
```
Comment (hrvojem) [inline doc/release-notes/1.0/1.0.10.rst]: Can you please provide more info (or ticket-IDs) on why/how this was implemented?

File: doc/release-notes/1.0/1.0.10.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+ * *Percona Agent* has improved the data sender timeout handling.
+
+ * New ``--status`` option has been implemented which can be used to get information on agent's status via API
+
+ * *Percona Agent* now enables the data compression when it gets installed.
+
+ * New ``--agent-mysql-user`` and ``--agent-mysql-pass`` options have been implemented to support installing agent with already existing user which has proper privileges on the server.
+
+ * *Percona Agent* now provides user-friendly error messages with additional details for troubleshooting, instead of general ``Error has occured`` message.
```
Comment (daniel-nichter) [inline doc/release-notes/1.0/1.0.10.rst]: This is a very broad statement, but the change is much more specific: better error messages just when connections to MySQL fail (ask @cfsalguero for more details).

File: doc/release-notes/1.0/1.0.10.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+ * *Percona Agent* now enables the data compression when it gets installed.
+
+ * New ``--agent-mysql-user`` and ``--agent-mysql-pass`` options have been implemented to support installing agent with already existing user which has proper privileges on the server.
+
+ * *Percona Agent* now provides user-friendly error messages with additional details for troubleshooting, instead of general ``Error has occured`` message. 
+ 
+ * *Percona Agent* now reports sender statistics for last 24 hours alongside with the last value.
+
+ * *Percona Agent* now sets up the percona agent user with `max_user_connections <https://dev.mysql.com/doc/refman/5.6/en/server-system-variables.html#sysvar_max_user_connections>`_ set to ``5`` to remove the possibility that *Percona Agent* could use up all the connections. This change only affects the new installations (unless the same limit is added manually for existing users).
```
Comment (daniel-nichter) [inline doc/release-notes/1.0/1.0.10.rst]: `*Percona Agent*` at start, `percona agent` a few words later?

File: doc/release-notes/1.0/1.0.10.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ 
+ * *Percona Agent* now provides better insight into the agent's server-side data spool by implementing the ``data-spooler-count``, ``data-spooler-size``, and ``data-spooler-oldest`` statuses.
+
+ * Normally agent status is only obtainable via API. If API or network is down, ``SIGUSR1`` signal can now be used by user to get agent status locally. 
+
+ * In case agent's primary websocket fails somehow, ``SIGHUP`` signal can be used to provide a local, manual override to reconnect without having to restart the whole agent. I.e. without interrupting the data collection.
```
Comment (daniel-nichter) [inline doc/release-notes/1.0/1.0.10.rst]: `s/whole //` and `s/. I.e. / and without /`

File: doc/release-notes/1.0/1.0.10.rst [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+ * The installation script did not work with ``--help`` option if the agent was already installed.
+
+ * InnoDB Stats were not re-enabled after MySQL server was restarted.
+
+ * Due to timeout error while closing the websocket, the agent could disconnect from the API without the API noticing it.
+ 
+ * In some cases ``USE database`` lines in the slow query log were not parsed correctly.
+
+ * *MySQL* restart monitor could send false-positive signals which could cause agent to unnecessary restart :ref:`query-analytics`.
```
Comment (daniel-nichter) [inline doc/release-notes/1.0/1.0.10.rst]: `s/unnecessary/unnecessarily/`

**Comment by daniel-nichter** [comment]
:+1:

**Pull request merged**